app/src/lib/stores/git-store.ts

```diff
@@ -923,6 +914,30 @@ export class GitStore extends BaseStore {
     return status
   }
 
+  /**
+   * Find a commit in the local cache, or load in the commit from the underlying
+   * repository.
+   *
+   * This will error if the commit ID cannot be resolved.
+   */
+  private async lookupCommit(sha: string): Promise<Commit> {
+    const cachedCommit = this.commitLookup.get(sha)
+    if (cachedCommit != null) {
+      return Promise.resolve(cachedCommit)
```

Comment:
I'm doing this here because I remember something about returning before the `await` being problematic, but that might have been before Chrome supported native promises, so this might be unnecessary. Gonna leave this here for safety.

Reply:
I’m pretty sure this will be fine.